src/storage/txn/txn_status_cache.rs

```diff
@@ -219,7 +261,10 @@ type TxnStatusCacheSlot =
 /// Note that the `TxnStatusCache` updates metrics in some operations assuming
 /// there's at most one instance of `TxnStatusCache` in a process.
 pub struct TxnStatusCache {
-    slots: Vec<CachePadded<Mutex<TxnStatusCacheSlot>>>,
+    // default cache for committed txns
+    normal_cache: Vec<CachePadded<Mutex<TxnStatusCacheSlot>>>,
+    // for large txns, or any txn whose min_commit_ts needs to be cached
```

Comment:
Because they actually serve different purposes. The design prevents large transactions from being evicted due to normal transactions. This is how it "prioritizes" large transactions.